cmd/family-migrator/main.go

```diff
@@ -206,7 +206,10 @@ func executePlan(kongCtx *kong.Context, planDir string, opts *Options) {
 				logger: logging.NewLogrLogger(zl.WithName("family-migrator")),
 			}))
 	} else {
-		planExecutor = migration.NewPlanExecutor(*plan, []migration.Executor{executor})
+		planExecutor = migration.NewPlanExecutor(*plan, []migration.Executor{executor},
```

Comment:
nit style comment: How about initializing an executor callback variable and instantiating the plan executor using that variable. This will prevent the if/else branch and prevent the duplicated `migration.NewPlanExecutor` code. So something like:
```go
cb := &loggerCallback{
				logger: logging.NewLogrLogger(zl.WithName("family-migrator")),
			}
if stepByStep {
   cb = &executionCallback{
				logger: logging.NewLogrLogger(zl.WithName("family-migrator")),
			}
}
planExecutor = planExecutor = migration.NewPlanExecutor(*plan, []migration.Executor{executor},
			migration.WithExecutorCallback(cb)))
```

Reply:
Done!